tests/test_power.py

```diff
@@ -62,5 +62,33 @@ def test_remove_board(self):
         # check the new board has gone
         self.assertTrue('ABC' not in self.robot.power_boards)
 
+    def test_buzz_args(self):
+        with self.assertRaises(ValueError):
+            self.robot.power_board.buzz(1, note='c', frequency=100)
+
+        with self.assertRaises(ValueError):
+            self.robot.power_board.buzz(1)
+
+        with self.assertRaises(KeyError):
```

Comment:
i'd be tempted to break the note validity checks out from the argument presence checks.

I'd also suggest adding tests which explore whether the case of the note matters.